src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -1649,6 +1649,28 @@ public static function provideRenderVerticalTests(): \Traversable
             $books,
         ];
 
+        yield 'With multibyte characters in some cells' => [
```

Comment:
Just to confirm, where are the multibytes characters in the headers ?

And the tests should be renamed: your fix is about characters in headers

Reply:
it's the `í` in `Dívíne` (I have clarified that in the fixture when renaming it)